src/app/components/Footer/index.jsx

```diff
@@ -1,27 +1,121 @@
 import React from 'react';
 import styled from 'styled-components';
 import {
-  C_STORM,
+  C_ORBIT_GREY,
   C_WHITE,
+  FF_NEWS_SANS_BLD,
```

Comment:
This is a non-trivial increase in page weight. Is it really worth it for the footer menu? I think the regular font with overridden font-weight would probably suffice.

Reply:
@benjaminhobbs bold text indicates to the user that it's a link, otherwise they have to guess what text is a link and what isn't